backend/src/main/java/com/hooria/todo/repository/CardRepository.java

```diff
@@ -0,0 +1,115 @@
+package com.hooria.todo.repository;
+
+import com.hooria.todo.domain.Card;
+import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import javax.sql.DataSource;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class CardRepository {
+
+    private final SimpleJdbcInsert insertJdbc;
+    private final NamedParameterJdbcTemplate jdbc;
+    private final RowMapper<Card> cardRowMapper;
+
+    public CardRepository(DataSource dataSource) {
+        jdbc = new NamedParameterJdbcTemplate(dataSource);
+        insertJdbc = new SimpleJdbcInsert(dataSource)
+            .withTableName("todo_card")
+            .usingGeneratedKeyColumns("id");
+
+        cardRowMapper = (rs, row) ->
+            new Card(
+                rs.getLong("id"),
+                rs.getInt("status"),
+                rs.getString("title"),
+                rs.getString("content"),
+                rs.getString("user_id"),
+                rs.getInt("appliance_info"),
+                rs.getObject("created_at", LocalDateTime.class),
+                rs.getObject("modified_at", LocalDateTime.class),
+                rs.getBoolean("deleted_yn"),
+                rs.getInt("index")
+            );
+    }
+
+    public List<Card> findAll() {
+        return jdbc.query(
+            "select id, status, title, content, user_id, appliance_info, created_at, modified_at, deleted_yn, index "
+                + "from todo_card where deleted_yn = false",
+            Collections.emptyMap(), cardRowMapper
+        );
+    }
+
+    public Optional<Card> findById(long id) {
+        SqlParameterSource namedParameters = new MapSqlParameterSource().addValue("id", id);
+        List<Card> result = jdbc.query(
+            "select id, status, title, content, user_id, appliance_info, created_at, modified_at, deleted_yn, index "
+                + "from todo_card where id = :id",
+            namedParameters, cardRowMapper
+        );
+        return result.stream().findAny();
```

Comment:
```suggestion
        Card card = DataAccessUtils.singleResult(jdbc.query(
                "select id, status, title, content, user_id, appliance_info, created_at, modified_at, deleted_yn, index "
                + "from todo_card where id = :id",
                namedParameters, cardRowMapper));

        return Optional.ofNullable(member);
```

요렇게! 해보는 것도 좋을 거 같아~~

Reply:
`DataAccessUtils.singleResult` 왓 이스 디스...?!?! 또 새로운 것을 배워갑니다.
선 채택 후 공부 갑니다.